### PR TITLE
Fix dynamic voting window

### PR DESCRIPTION
**Pull request by PlasmaPower**
Currently it crashes the server. I've also switched to arrow for anything datetime, as that's the standard for the project and it's a lot better than some hand rolled parsing.

### CODE REVIEW

File: github_api/voting.py [hunk 1]
```diff
@@ -206,14 +205,13 @@ def get_initial_voting_window(now):
     return seconds
 
 
-def get_extending_voting_window(api, urn):
+def get_extended_voting_window(api, urn):
```
Comment (PlasmaPower) [inline github_api/voting.py:208]: This is the part that was crashing it.

File: github_api/repos.py [hunk 2]
```diff
@@ -29,6 +29,4 @@ def set_desc(api, urn, desc):
 def get_creation_date(api, urn):
     """ returns the creation date of the repo """
     data = api("get", get_path(urn))
-    # this is the field for watchers.  do not be tricked by "watchers_count"
-    # which always matches "stargazers_count"
-    return datetime.strptime(data["created_at"], "%Y-%m-%dT%H:%M:%SZ").date()
```
Comment (PlasmaPower) [inline github_api/repos.py:34]: Also, how in the world did the PR get through with this comment? Did no one read the changes?

Comment (flibustier) [reply github_api/repos.py:34]: Outch… sorry :/

**Comment by flibustier** [comment]
Thank you! Sorry for the mess :/

**Comment by mark-i-m** [comment]
So how does this get merged in?

**Comment by PlasmaPower** [comment]
@mark-i-m the maintainer of chaosbot merges it.

**Comment by amoffat** [comment]
Ok it's been merged in, thanks for the hotfix @PlasmaPower

**Pull request merged**